### ISSUE TITLE
Can't see projects, uploaded files and partially cannot upload a .psd

### ISSUE THREAD

**Issue by olixem** [avocode/avocode#944]
IMPORTANT: Please try to describe how did it happen.

+ If I create an empty project I cannot upload a .psd through the dialog box the comes after. I have to go back, and upload a .psd that's unassigned to a project
+ If I upload a .psd through metod described above and assign it to a project I can't see it. My project & design count are both 0. I can view the uploaded project, but when I close it, I can't find it neither in projects nor in designs and the only way to view it again is to reupload it.
-----
App v.2.8.2
Source UID: 72431
```
TypeError: Cannot read property 'get' of null
    at Function.e.fromProject (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:14:15014)
    at e.createUploadTargetFromProject (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:14:3211)
    at Constructor._createUploadTarget (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:34:18328)
    at Constructor._handleUploadNewDesign (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:34:18505)
    at Constructor._handleUploadNewDesign (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:36:4098)
    at Object.invokeGuardedCallback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactErrorUtils.js:27:12)
    at executeDispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginUtils.js:79:21)
    at Object.executeDispatchesInOrder (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginUtils.js:99:7)
    at executeDispatchesAndRelease (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginHub.js:43:22)
    at executeDispatchesAndReleaseTopLevel (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginHub.js:54:10)
    at Array.forEach (native)
    at forEachAccumulated (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/forEachAccumulated.js:23:9)
    at Object.processEventQueue (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginHub.js:259:7)
    at runEventQueueInBatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactE
```

**Comment by petrbrzek** [comment]
Thanks for the report. We extended the subscription of your team so you should be able to use Avocode. We'll fix the issue in the next version.

**Comment by petrbrzek** [comment]
Hopefully fixed in 2.9.0.
